guides/common/modules/proc_upgrading-project-from-el7-to-el8.adoc

```diff
@@ -1,15 +1,47 @@
-[id="Upgrading-Project-from-EL7-to-EL8_{context}"]
+[id="Upgrading-Project-from-Enterprise-Linux-7-to-Enterprise-Linux-8_{context}"]
+ifdef::foreman-el,katello[]
 = Upgrading {Project} from Enterprise Linux 7 to Enterprise Linux 8
+endif::[]
+ifdef::satellite[]
+= Upgrading {Project} from {RHEL} 7 to {RHEL} 8
+endif::[]
 
+ifdef::foreman-el,katello[]
 Use this procedure to upgrade your {Project} installation from Enterprise Linux 7 to Enterprise Linux 8.
 LEAPP is the supported tool for Enterprise Linux 7 to Enterprise Linux 8 upgrades.
```

Comment:
Now that you dropped this line in the Satellite case, I realize it's also not really necessary here (but obviously not blocking)